pkg/webreg/webreg.go

```diff
@@ -613,12 +613,23 @@ Example:
 <p>
 All images built for the release targeted by the job (ex:
 <code>release-4.2</code>) are available under <code>stable:tag-name</code>
-(example: <code>stable:installer</code> or <code>stable:cli</code>). There
```

Comment:
Since we're changing the paragraph, I think it's worth noting that this corresponds to `tag_specification`.